frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -349,6 +394,27 @@ export default {
     content: '';
   }
 }
+// Duration-Display
+.e-picasso-entry__duration {
+  pointer-events: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
why do we need all four directions, and why are they all set to 0?
https://developer.mozilla.org/en-US/docs/Web/CSS/top#try_it

Reply:
@BacLuc yes, otherwise they are set to auto and ignored. But we could replace those lines with:
```suggestion
  inset: 0;
```